includes/admin.php

```diff
@@ -34,6 +37,51 @@ function add_meta_box() {
 	);
 }
 
+/**
+ * Enqueue scripts & styles.
+ */
+function enqueue_scripts() {
+	if ( get_current_screen()->post_type !== Redirects_Post_Type\SLUG ) {
+		return;
+	}
+
+	wp_enqueue_style( 'hm-redirects-admin', plugins_url( '/assets/admin.css', dirname( __FILE__ ) ), [], null, 'screen' );
+}
+
+/**
+ * Filter the posts listing columns.
+ *
+ * @param array $columns List of column ids and labels.
+ * @return array
+ */
+function filter_posts_columns( array $columns ) : array {
+	$columns = [
+		'title' => __( 'From', 'hm-redirects' ),
+		'to' => __( 'To', 'hm-redirects' ),
+		'status' => __( 'Status', 'hm-redirects' ),
+		'date' => __( 'Date' ),
+	];
+	return $columns;
+}
+
+/**
+ * Output for the custom admin columns.
+ *
+ * @param string $column Current column ID.
+ * @param int    $post_id Current post ID.
+ */
+function posts_columns_content( string $column, int $post_id ) {
+	$post = get_post( $post_id );
+
+	if ( $column === 'to' ) {
+		echo esc_html( sanitize_text_field( $post->post_excerpt ) );
```

Comment:
By using `sanitize_text_field()` you run into the same issue as was present in the matching of non ASCII URLs bug. Special characters get octet encoded, and these get stripped during sanitisation.

Beyond this, we are outputting data, so there should be no sanitisation applied.